frontend/src/scenes/billing/Billing.tsx

```diff
@@ -152,13 +159,13 @@ export function Billing(): JSX.Element {
                 </LemonBanner>
             ) : null}
 
-            {!billing?.has_active_subscription && !billing?.trial && platformAndSupportProduct && (
+            {!isManagedAccount && !billing?.has_active_subscription && !billing?.trial && platformAndSupportProduct && (
```

Comment:
This change probably conflicts with your other PR - also managed accounts shouldn't be on free accounts so I think this won't be needed.

Reply:
> This change probably conflicts with your other PR
Yup will see what gets merged first and resolve conflicts there

> also managed accounts shouldn't be on free accounts so I think this won't be needed.
I assumed so too initially but then thought that maybe an account gets AE/CSM assigned in Vitally when they reach out and start convo but before they sign. Will confirm with Mine.

Reply:
Simon replied

> Q: Can an account be managed (get AE/CSM assigned in Vitally) before becoming a paying customer?

> A: Yes they might do if the AE is managing them through a trial

> Q: Got it and technically could they already get AE/CSM assigned before starting a trial? E.g. when they reach out to the team as a prospect. Or AE/CSM would only get assigned once at least on trial?

> A: We’d only assign them in Vitally once having spoken with them and got them on a trial

So technically it's not needed since we'd hide this already if they're on the trial but I'd still leaning towards keeping this check to make it future proof in case sales process changes?

Reply:
We do have cta in other places so it's not really hiding all of them but I'm ok with this change for now.

Reply:
Got it, will keep it then and try to track down other places where we might want to hide them